src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
I'm for the `delete` route as well. I don't think we'd need the guard as it shoudl just be a no-op there.